breakerbp/failure_ratio.go

```diff
@@ -26,32 +26,32 @@ type FailureRatioBreaker struct {
 // Config represents the configuration for a FailureRatioBreaker.
 type Config struct {
 	// Minimum requests that need to be sent during a time period before the breaker is eligible to transition from closed to open.
-	MinRequestsToTrip int
+	MinRequestsToTrip int `yaml:"primary"`
```

Comment:
why is this "primary"?

Reply:
Ideally this can be mapped to the actual field name, e.g.
```go
`yaml:"minRequestsToTrip"`
```

Reply:
Thats was a miss. I agree `yaml:"minRequestsToTrip"` is the correct value.
- [ ] Fix min requests tag